src/components/TextField/TextField.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ThemeProvider } from '@material-ui/core';
+
+import Component from './TextField';
+import theme from 'theme';
+
+export default {
+    title: 'Components/TextField',
+    component: Component,
+};
+
+export function TextField() {
+    return (
+        <ThemeProvider theme={theme}>
```

Comment:
What errors were happening that this was required?  I'm curious because I already, by default, wrap all storybooks in the theme provider.

Reply:
Removed, I thought this was part of the boiler plate for stories